drivers/iio/proximity/vcnl3020.c

```diff
@@ -0,0 +1,338 @@
+// SPDX-License-Identifier: GPL-2.0-only
+/*
+ * vcnl3020.c - Support for Vishay VCNL3020 proximity sensor
+ *
+ * based on vcnl4000.c
+ */
+
+#include <linux/module.h>
+#include <linux/i2c.h>
+#include <linux/err.h>
+#include <linux/delay.h>
+
+#include <linux/iio/iio.h>
+#include <linux/iio/sysfs.h>
+#include <linux/platform_device.h>
+#include <linux/iio/proximity/vcnl3020.h>
+
+static const struct i2c_device_id vcnl3020_id[] = {
+	{ "vcnl3020", 0 },
+	{}
+};
+MODULE_DEVICE_TABLE(i2c, vcnl3020_id);
+
+static int32_t vcnl3020_init(struct vcnl3020_data *data)
+{
+	s32 rc;
+	u32 proximity_rate, led_current, threshold, count_exceed, param_val;
```

Comment:
Don't do declarations in that way

Reply:
and which it should be done then?

Reply:
u32 proximity_rate;
u32 led_current;
... and so on

Reply:
https://elixir.bootlin.com/linux/latest/source/kernel/async.c#L114

it is not prohibited also linux kernel guideline doesn't prohibit.

Reply:
It is generally better to have one declaration per line as it simplifies review of future changes. Added/removed text in the middle of a line is harder to review than added/removed lines.

Reply:
okay, I can make it per line, both variants are fine.